src/views/JoinHouse.vue

```diff
@@ -76,7 +80,9 @@ methods: {
     },
 
     joinHouse(){
-        this.$store.dispatch("joinHouse", {email: this.$store.state.user.email, roomkey: this.roomkey})
+        if(this.roomkey === 32){
+            this.$store.dispatch("joinHouse", {email: this.$store.state.user.email, roomkey: this.roomkey})
+        } 
```

Comment:
Where's the else catch here?

Reply:
I mean, should we have an else catch here?

Reply:
Good catch! Added error message!